framework/src/modules/chain/dpos/delegates_info.js

```diff
@@ -212,13 +228,20 @@ class DelegatesInfo {
 
 			const delegateAccounts = await this.storage.entities.Account.get(
 				{ publicKey_in: summedRound.delegates },
-				{},
+				{ extended: true },
```

Comment:
Do we really need `extended` here? :)

Reply:
Yes. `votedDelegatesPublicKeys` is only returned when using `extended: true`